snuba/web/split.py

```diff
@@ -328,6 +329,17 @@ def execute(
             metrics.increment("column_splitter.query_above_limit")
             return None
 
+        # Do not split if there is already a = or IN condition on an ID column
+        for expr in query.get_condition_from_ast() or []:
+            match = FunctionCall(
+                None,
+                Or([String(ConditionFunctions.EQ), String(ConditionFunctions.IN)]),
+                (Column(String(self.__id_column)), AnyExpression(),),
```

Comment:
`Column(None, None, String(self.__id_column))`. the parameters of the column match are in the same order as for the AST column. The first attribute is the alias, the second ois the table and the third is the column name.

I guess the test still works because we generally provide an alias to columns that correspond to the column name.